anagrafica/management/commands/valorizza_email_servizio.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+from django.core.management.base import BaseCommand
+
+from anagrafica.models import Persona
+
+
+class Command(BaseCommand):
```

Comment:
Mi aspetterei una migrazione per la valorizzazione iniziale. Perche' un comando? Va eseguito manualmente nuovamente in futuro? Se si, in quali circostanze?

Reply:
pensavo non fosse da escludere l'ipotesi di rilanciarlo sì. Direi per riordinare le mail, cambiassero i parametri (per esempio l'altro commento in cui non solo *.cri.it è di servizio). Però se vuoi la porto in migration